src/shader/cel.rs

```diff
@@ -78,7 +78,15 @@ impl<'a> Pipeline for CelShader<'a> {
             intensity if intensity > 0.05 => color * 0.35,
             _ => color * 0.1,
         };
+
+        // Gamma correction
+        let gamma = 2.2;
```

Comment:
small nit: leave a comment with the link `https://learnopengl.com/Advanced-Lighting/Gamma-Correction` or a description so that future readers don't have to read this PR to understand where 2.2 came from (i.e. there are other values like 2.35/2.4 that you could have chosen from a brief google on gamma correction)

Reply:
Thanks! Done!